apps/nextjs-website/react-components/components/PressRelease/PressRelease.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {
+  Box,
+  Typography,
+  Container,
+  Stack,
+  Link,
+  useTheme,
+} from '@mui/material';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+
+import {
+  TextColor,
+  ExtraTextColor,
+  BackgroundColor,
+} from '../common/Common.helpers';
+
+const PressRelease = ({
+  eyelet,
+  title,
+  subtitle,
+  body,
+  ctaText,
+  ctaHref,
+  sectionID,
+}: {
+  eyelet: string;
+  title: string;
+  subtitle?: JSX.Element;
+  body: JSX.Element;
+  ctaText: string;
+  ctaHref: string;
+  sectionID: string;
+}) => {
```

Comment:
```suggestion
}: PressReleaseProps) => {
```
Use already available interface / type